lk2nd/device/dts/msm8952/msm8937-huawei-aum.dts

```diff
@@ -0,0 +1,67 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <skeleton64.dtsi>
+#include <lk2nd.dtsi>
+
+/*
+ * To flash lk2nd onto the device, you need to flash lk2nd as the
+ * kernel and flash an empty image in place of the ramdisk.
+ * To create an empty image use the following command:
+ * mkbootimg --kernel /dev/null --ramdisk /dev/null --base 0x80000000 --kernel_offset 0x00008000 --ramdisk_offset 0x02000000 --second_offset 0x00f00000 --tags_offset 0x00000100 --pagesize 2048 --header_version 0 -o ramdisk.img"
```

Comment:
ugh do they really use two bootimg files for kernel and ramdisk? what if you just completely ignore the ramdisk partition and keep whatever downstream had in it?

Do I understand correctly that lk2nd.img can be just flashed to "kernel"?

Reply:
It can be just flashed to "kernel." I just am not sure if it won't cause anything if there's that ramdisk partition there, and trying to erase it just throws you into eRecovery (TWRP in my case).